aws/resource_aws_instance.go

```diff
@@ -283,7 +283,7 @@ func resourceAwsInstance() *schema.Resource {
 			},
 
 			"ebs_block_device": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
I am personally not aware of a good way to migrate the Terraform state from `TypeSet` to `TypeList`. When we are running into this situation its easier to implement something similar to #4117. A lot of investigative work has been done in this regard, see also #1099 and #1476.

Reply:
Got it--so we'd add an `ordered_ebs_block_device` field using `TypeList` that `ConflictsWith` the existing field? But if that's the right way to do this, then users are going to have to update their configs to get modifiable inline ebs volumes--and if they're going to have to update anyway, maybe it's better just to recommend using separate `aws_ebs_volume` resources instead. WDYT?